tools/protodoc/protodoc.py

```diff
@@ -134,6 +135,13 @@ def FormatHeader(style, text):
   return '%s\n%s\n\n' % (text, style * len(text))
 
 
+def FormatHeaderFromFile(style, file_level_comment, alt):
+  m = re.search('protodoc-title:\s([^\n]+)\n', file_level_comment)
+  if m is not None:
```

Comment:
Nit: In this case, I would just write: `if m:`. There are places you want to distinguish `None` and things like empty lists in Python, but this isn't one.